app/src/main/java/com/example/ex3/QuotesService.java

```diff
@@ -0,0 +1,53 @@
+package com.example.ex3;
+
+import android.app.AlarmManager;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.os.IBinder;
+import android.os.SystemClock;
+import android.widget.Toast;
+
+import androidx.core.app.NotificationCompat;
+
+public class QuotesService extends Service {
```

Comment:
its totally fine but you could use intentservice which is a bit easier and nicer to use (: